src/ts/demofactory.ts

```diff
@@ -118,6 +119,8 @@ export namespace DemoFactory {
         new Container({
           components: [
             new PlaybackToggleButton(),
+            new QuickSeekButton({ direction: QuickSeekDirection.Rewind, seekSeconds: 10 }),
+            new QuickSeekButton({ direction: QuickSeekDirection.Forward, seekSeconds: 10 }),
```

Comment:
Would it make sense to have these buttons as part of the default UI? I think they could be a good addition :)

Reply:
I think it would. However, changing anything in the default UI is not something taken too lightly, and internal discussions around this are planned. That's why for now we export the Button so it can be easily used for custom UIs and externally managed UIs, and only added it to the additional demo UI in the playground.

Reply:
Makes sense 👍🏼